addons/csw/CfgWeapons.hpp

```diff
@@ -0,0 +1,260 @@
+            assembleTo = QGVAR(staticHMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 840;
+        };
+        
+        displayName = CSTRING(StaticHMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticAutoHMGCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticAutoHMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 840;
+        };
+        
+        displayName = CSTRING(StaticAutoHMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticHMGTallCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticHMGTallWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 840;
+        };
+        
+        displayName = CSTRING(StaticHMGTallBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    
+    class GVAR(staticGMGCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticGMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 780;
+        };
+        
+        displayName = CSTRING(StaticGMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticAutoGMGCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticAutoGMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 780;
+        };
+        
+        displayName = CSTRING(StaticAutoGMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticGMGTallCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticGMGTallWeapon);
```

Comment:
For high/low the Tripods are different, not the weapons
will get a little messy, can worry about this later